pymemcache/__init__.py

```diff
@@ -1,2 +1,5 @@
 __version__ = '2.0.0'
 __author__ = "Charles Gordon"
+
+from pymemcache.client.base import Client  # noqa
+from pymemcache.client.base import PooledClient  # noqa
```

Comment:
Let's commit these lines as part of their own PR (#197) and focus just on fixing the `six` error here. Can you remove these from this PR?